stable/mssql-linux/templates/deployment.yaml

```diff
@@ -52,3 +61,26 @@ spec:
       nodeSelector:
 {{ toYaml .Values.nodeSelector | indent 8 }}
     {{- end }}
+      volumes:
+      - name: data
+      {{- if .Values.persistence.enabled }}
+        persistentVolumeClaim:
+          {{- if .Values.persistence.existingDataClaim }}
+          claimName: {{ .Values.persistence.existingDataClaim }}
+          {{- else }}
+          claimName: {{ template "mssql.fullname" . }}-data
+          {{- end -}}
+      {{- else }}
+        emptyDir: {}
+      {{- end }}
+      - name: transactionlog
+      {{- if .Values.persistence.enabled }}
+        persistentVolumeClaim:
+          {{- if .Values.persistence.existingLogClaim }}
+          claimName: {{ .Values.persistence.existingLogClaim }}
+          {{- else }}
+          claimName: {{ template "mssql.fullname" . }}-translog
```

Comment:
Referenced PVC claim name -translog does not match actual PVC name -log

Reply:
Thanks for pointing this out.  Corrected